SPECS-EXTENDED/ocaml-base/ocaml-base.spec

```diff
@@ -8,17 +8,17 @@ Distribution:   Mariner
 %global srcname base
 
 Name:           ocaml-%{srcname}
-Version:        0.14.1
-Release:        2%{?dist}
+Version:        0.15.0
+Release:        1%{?dist}
 Summary:        Jane Street standard library for OCaml
 
 License:        MIT
 URL:            https://opensource.janestreet.com/base/
 Source0:        https://github.com/janestreet/%{srcname}/archive/v%{version}/%{srcname}-%{version}.tar.gz
-
+Patch0:         %{name}-ocaml413compat.patch
 BuildRequires:  ocaml >= 4.07.0
```

Comment:
Shouldn't we require ocaml >= 4.13 due to compat patch then?

Reply:
The compat patch does not prevent older versions from working, it merely enables newer versions to work.